configs/prod/ethereum/p2p.json

```diff
@@ -1,128 +1,129 @@
 {
-    "common": {
-        "delegation_registry": {
-            "abi_key": "8f910c9aa38cffa69fc4a97a81cbc79dda306e2a",
-            "address": "0x00000000000000447e69651d841bD8D104Bed493",
-            "contract": "DelegationRegistry",
-            "properties_addresses": {}
-        },
-        "p2p_controller": {
-            "abi_key": "8c9e1e93f39f6d62e9d7d345eb17433df2fc0343",
-            "address": "0xa11704B3dD545dC0771d9a9c5572064Db2A5C062",
-            "contract": "P2PLendingControl",
-            "properties": {
-                "trait_roots_key": "configs.trait_roots"
-            },
-            "properties_addresses": {}
-        }
+  "common": {
```

Comment:
the formatting is a bit messed up, can we keep the original to avoid merge issues?

Reply:
Done on my last commit @cfcfs . My editor had defined for json files tab size to 2.